sky/core.py

```diff
@@ -740,19 +742,20 @@ def spot_queue(refresh: bool,
         stop_msg = 'To view the latest job table: sky spot queue --refresh'
     controller_status, handle = spot.is_spot_controller_up(stop_msg)
 
-    if controller_status is None:
-        return []
```

Comment:
Why deleting this?

Reply:
We should raise the `ClusterNotUp` exception for the case where the controller does not exist as well to be more consistent with the other cases. That will make it easier for the caller to find out the correct status of teh controller.

Reply:
Done. Thanks!